index.html

```diff
@@ -1,19 +1,87 @@
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
-<body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+<body>
+    <header>
+        <img src="./img/karma-logo.svg" alt="Karma Logo">
+        <nav>
+            <ul>
+                <li>
+                    <a href="#"><strong>Meet Karma</strong></a>
+                </li>
+                <li>
+                    <a href="#">How it Works</a>
+                </li>
+                <li>
+                    <a href="#">Store</a>
+                </li>
+                <li>
+                    <a href="#">Blog</a>
+                </li>
+                <li>
+                    <a href="#">Help</a>
+                </li>
+                <li>
+                    <a href="#">Login</a>
+                </li>
+            </ul>
+        </nav>
+        <div class="hamburger-menu-wrapper">
+            <div class="hamburger-menu"></div>
+            <div class="hamburger-menu"></div>
+            <div class="hamburger-menu"></div>
+        </div>
+    </header>
+    <main>
+        <section class="hero">
+            <h1>Introducing Karma</h1>
+            <p>Bring WiFi with you, everywhere you go.</p>
+            <button>Learn More</button>
+        </section>
+        <section class="karma-details">
+            <h2>Everyone needs a little Karma</h2>
+            <div class="row-container">
+                <div class="column-container">
+                    <img src="./img/icon-devices.svg" alt="Image outline of devices that can connect to the internet">
+                    <p>Internet for all devices</p>
+                </div>
+                <div class="column-container">
+                    <img src="./img/icon-coffee.svg" alt="Image outline of a steaming cup of coffee">
+                    <p>Boost your productivity</p>
+                </div>
+                <div class="column-container">
+                    <img src="./img/icon-refill.svg" alt="Image outline of a gas pump">
+                    <p>Pay as you go</p>
+                </div>
+            </div>
+        </section>
+        <section class="feature">
+            <img src="./additional/feature-img.png">
+            <div class="feature-wrapper">
+                <p><em>Wherever I am, I just don't worry about my connection anymore!</em></p>
+                <button>Get Karma Today</button>
+            </div>
+        </section>
+        <hr>
+        <footer>
+            <h3>Join us on</h3>
+            <a href="#"><img src="./img/twitter-icon.svg" alt="Twitter Logo"></a>
+            <a href="#"><img src="./img/facebook-icon.svg" alt="Facebook Logo"></a>
+            <a href="#"><img src="./img/instagram-icon.svg" alt="Instagram Logo"></a>
```

Comment:
An SVG is a vector image which uses mathematical points to create an image, they are scaleable without losing its quality and can also have their colour changed. They are very good for responsive apps or websites where the size always changes. Images are raster files which are made up of pixels, although they can look high quality at first, they usually change quality when the size changes because the size of the pixels change, they are good for more static pages and digital prints.